common/utilities/isBrowser.js

```diff
@@ -0,0 +1 @@
+export const isBrowser = () => process.browser
```

Comment:
id rename the file to `browser` or `browserChecks` `isServer` is an obvious addition in future